packages/connections/src/components/connection-list/connection-menu.tsx

```diff
@@ -100,10 +100,22 @@ function ConnectionMenu({
         type: 'show-success-toast',
       });
     } catch (err) {
-      dispatch({
-        type: 'show-warning-toast',
-        error: err,
-      });
+      if (err instanceof Error) {
+        dispatch({
+          type: 'show-warning-toast',
+          error: err.message,
+        });
+      } else if (typeof err === 'string') {
+        dispatch({
+          type: 'show-warning-toast',
+          error: err,
+        });
+      } else {
+        dispatch({
+          type: 'show-warning-toast',
+          error: 'An error occured when copying to keybaord. Please try again.',
```

Comment:
```suggestion
          error: 'An error occurred when copying to keyboard. Please try again.',
```

Reply:
The double typo 💀 
Just installed a spellchecker extension in vscode xD